internal/ctrlutils/ingress-status.go

```diff
@@ -137,7 +137,7 @@ func UpdateIngressV1(ctx context.Context, logger logr.Logger, svc file.FService,
 		routeInf := strings.Split(*((*route).Name), ".")
 		namespace := routeInf[0]
 		name := routeInf[1]
-		log.Infof("updating status for v1.Ingress route: name %s namespace %s", name, namespace)
+		log.Debugf("updating status for v1.Ingress route: name %s namespace %s", name, namespace)
```

Comment:
Our logging library (logr) [supports structured logging](https://pkg.go.dev/github.com/go-logr/logr#pkg-overview). Therefore our logs shouldn't try to format-string the log message, but instead put values such as name and namespace as entry fields:

```suggestion
		log.Debugf("updating status for v1.Ingress route", "name", name, "namespace", namespace)
```